plugin/bind/OWNERS

```diff
@@ -0,0 +1,4 @@
+reviewers:
+  - miekg
```

Comment:
I can step-up for reviewers on bind.
I am currently modifying to accept several addresses. After that change it will be a specific plugin.

Reply:
I am ok for others where I am involved (k8s, proxy)